validation/validation_test.go

```diff
@@ -88,13 +88,53 @@ func TestValidateBasic(t *testing.T) {
 	assert.Nil(t, runtimeInsideValidate(g))
 }
 
+// Test whether rootfs Readonly can be applied as false
+func TestValidateRootFS(t *testing.T) {
+	g := getDefaultGenerator()
+	g.SetRootReadonly(false)
+
+	assert.Nil(t, runtimeInsideValidate(g))
+}
+
+// Test whether rootfs Readonly can be applied as true
+func TestValidateRootFS(t *testing.T) {
+	g := getDefaultGenerator()
+	g.SetRootReadonly(true)
+
+	assert.Nil(t, runtimeInsideValidate(g))
+}
+
+// Test whether hostname can be applied or not
+func TestValidateHostname(t *testing.T) {
+	g := getDefaultGenerator()
+	g.SetHostname("hostname-specific")
+
+	assert.Nil(t, runtimeInsideValidate(g))
+}
+
+// Test whether mounts are correctlt mounted
```

Comment:
“correctlt” → “correctly”.  Or just use `// Validate mounts`, or leave the comments off?  `TestValidateMounts` sounds pretty clear to me even without a comment.